tests/test_fairworkflow.py

```diff
@@ -1,65 +1,58 @@
-import pytest
-import requests
-
 from fairworkflows import FairWorkflow, FairStep, fairstep
 
-def test_build_fairworkflow():
-    workflow = FairWorkflow(description='This is a test workflow.')
 
-    assert(workflow is not None)
-    assert(workflow.description() is not None)
+class TestFairWorkflow:
+    test_description = 'This is a test workflow.'
+    workflow = FairWorkflow(description=test_description)
 
     step1 = FairStep(uri='http://www.example.org/step1')
     step2 = FairStep(uri='http://www.example.org/step2')
     step3 = FairStep(uri='http://www.example.org/step3')
 
     workflow.add(step2, follows=step1)
     workflow.add(step3, follows=step2)
-
-    assert(workflow.validate() is False)
```

Comment:
I think it's worth also checking that the workflow (correctly) does not validate before any steps have been added or a first step set. What do you think?

Reply:
I do this in `test_build`. But I was struggling a bit with the fact that basically for every test we are building a workflow... I guess there is a way to fix this properly, but I am not sure if we should invest time in that.

Reply:
Yes, sorry, I meant we could have the check also before any steps are added. That way it also gets us more test coverage in the validate() function. Yeah, for now we need to build a new workflow before each test, but once I add in the functionality to load workflows from rdf (either remote or local) then we will be able to test on a range of pre-written workflows instead. That should get rid of this particular problem.

Reply:
Ah, great! 

I have added an assert for a workflow without steps!